### DISCUSSION TITLE
The xml files next to the assemblies

### DISCUSSION THREAD

**Discussion by ghost** [pascalabcnet/pascalabcnet#2949] [General]
Can PascalABC.NET IDE see and use them?

https://stackoverflow.com/questions/4015018/whats-the-purpose-of-xml-files-next-to-the-dll

**Comment by ghost** [comment]
The xml files are used for displaying documentation. It seems PascalABC.NET doesn't make use of them because I found the code completion of it doesn't show documentation, only methods and class names.

@ibond84

**Reply by ghost** [reply]
@miks1965

**Reply by ghost** [reply]
@SunSerega

**Reply by samuraiGH** [reply]
show your dll and xml files

**Comment by samuraiGH** [comment]
Yes
